config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -3,6 +3,8 @@ en:
     assessment:
       submission:
         submissions:
+          no_answer:
+            There's no answer for this question. The question could be added after this submission.
```

Comment:
There is no submitted answer for this question - this might be caused by the addition of this question after the submission is submitted.